ReactCommon/react/renderer/uimanager/primitives.h

```diff
@@ -30,13 +30,21 @@ struct ShadowNodeWrapper : public jsi::HostObject {
   ShadowNodeWrapper(SharedShadowNode shadowNode)
       : shadowNode(std::move(shadowNode)) {}
 
+  // The below method needs to be out-of-line in order for the class to have
+  // at least one "key function" (see https://itanium-cxx-abi.github.io/cxx-abi/abi.html#vague-vtable)
+  virtual ~ShadowNodeWrapper();
```

Comment:
can we use `= default` here instead/provide body right in the header? Or "key function" definition requires a body to be present?

Reply:
apparently, it needs to not be inlined and exists in one of the binaries to be treated a s a key function